ideahacks/helpers/userFunctions.js

```diff
@@ -0,0 +1,31 @@
+const formatUser = user => {
+  if (user.hasTeam) {
+    user.hasATeam = true
+  }
+  user.hasNoTeam = true
+  if (user.vehicleNeed) {
+    user.doesNeedVehicle = true
+  }
+  user.doesNotNeedVehicle = true
+
+  let emailSize = user.teammates.length
+  if (emailSize === 0) {
+    return user
+  }
+
+  switch (emailSize) {
```

Comment:
Proud of you dude. This is a pretty sick design and usage of a switch statement. Only suggestion I would have is that instead of checking if numberOfTeammates === 0, you should remove that if statement and just put a `default` case in your switch statement, and all it does it `break`. Seems cleaner this way, instead of terminating the function early.